examples/seismic/acoustic/acoustic_example.py

```diff
@@ -62,6 +63,15 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
     return summary.gflopss, summary.oi, summary.timings, [rec, u.data]
 
 
+@pytest.mark.parametrize('ndim', [1, 2, 3])
+def test_isoacoustic_stability(ndim):
+    shape = tuple([11]*ndim)
+    spacing = tuple([20]*ndim)
+    _, _, _, [rec, _] = run(shape=shape, spacing=spacing, tn=20000.0, nbl=0)
+    norm = lambda x: np.linalg.norm(x.data.reshape(-1))
+    assert np.isfinite(norm(rec))
```

Comment:
should we also not assert against some pre-computed value? a value around 10**7 is finite, but ...

Reply:
wouldn't really matter. That's ~10k timesteps it's either finite or NaN if unstable so `isfinite` i all needed to know it's stable.

Reply:
ahhhh right fair point